deployment-templates/deploy-forseti.yaml.in

```diff
@@ -84,15 +86,8 @@ resources:
     cloudsqlproxy-os-arch: linux.amd64
     db-port: 3306
 
-# Service accounts
-- name: forseti-gcp-reader
-  type: service_acct.py
-  properties:
-    accountId: {SERVICE_ACCT_GCP_READER}
-    displayName: forseti-gcp-reader
-
-- name: forseti-gsuite-reader
-  type: service_acct.py
-  properties:
-    accountId: {SERVICE_ACCT_GSUITE_READER}
-    displayName: forseti-gsuite-reader
+    # Forseti run frequency
+    # WARNING: only change this value if you know exactly what you're doing!
+    # This is a crontab schedule. The default is "run at minute 0", i.e. run
+    # Forseti at minute 0 of every hour.
+    run-frequency: "0 * * * *"
```

Comment:
Can we randomize the minute so not every forseti user is hammering the APIs at the top of every hour?

Also, should we run every other hour, or every 4th hour, is hourly needed for the average customer?

Reply:
I don't think hourly is necessary (especially now that inventory has more resources to crawl, the inventory building will take longer). If the customer needs something more frequent than every other hour, they need to wait for when we implement an event-driven inventory.

I'll change this to generate some number between 0 and 59 for the minute, and have it run every other hour.